src/14_cal.py

```diff
@@ -29,4 +29,50 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+curr_datetime = datetime.today()
+curr_month = curr_datetime.month
+curr_year = curr_datetime.year
+
+if(len(sys.argv))==1:
+    print(calendar.monthcalendar(curr_year,curr_month))
+elif (len(sys.argv) >=3):
+    if(len(sys.argv[2])!=4):
+        print("Year should be in the format 'YYYY'")
+    else:
+        try: 
```

Comment:
I like that you're experimenting with exceptions and exception handling here. Knowing how to catch them is pretty much mandatory. Maybe also look into throwing them yourself too? Can be really handy when it doesn't make sense to resolve the error locally.